net/acme/Makefile

```diff
@@ -9,10 +9,10 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=acme
 PKG_VERSION:=2.8.6
-PKG_RELEASE:=1
+PKG_RELEASE:=1.1
```

Comment:
PKG_RELEASE is just an integer, so just bump this to '2'

Reply:
Oops, will amend